### ISSUE TITLE
More verbose readme?

### ISSUE THREAD

**Issue by dkoslicki** [medvedevgroup/mutation-rate-intervals#1]
Currently, to figure out what each function does, you need to consult the `--h` help text or in the code comments for scripts without `--h`. Might be helpful to have a brief description of each file. Eg. what's the difference between `simulate_unit_errors_v1.py` and `simulate_unit_errors.py`

**Comment by rsharris** [comment]
(The email I receive this on was out of service from Friday afternoon until this morning)

Those are good points. I'll take some time later today to improve the readme.

**Comment by rsharris** [comment]
The readme has been updated, adding usage information for all the command line programs.

The "_v1" in module names has been changed to "_thm5".

The previously existing note in the readme (in "Repo organization") briefly explains the difference between those v1 modules and the slicer modules. I decided against elaborating that note because the only people who would use the simulation are those trying to reproduce the results from the manuscript (i.e. not the typical end user).
